lib/src/core/base/shared_preference/shared_pref_key.dart

```diff
@@ -1,2 +1,2 @@
-const String USER_UID = 'USER_UID';
-const String IS_LOGGED_IN = '';
\ No newline at end of file
+const String USER_UID = '';
+const bool IS_LOGGED_IN = false;
```

Comment:
these are keys for setting the shared prefs so they shouldn't have a value like false. It should be like this:

```
const String USER_UID = 'USER_UID';
const String IS_LOGGED_IN = 'IS_LOGGED_IN';
```

Then you can replace all the places where you hardcoded these string to use these keys.

Reply:
ok shared_pref_key.dart is only for keys, values for selected key should be getted by SharedPref.getStringAsync(KEY_NAME_EXAMPLE)